.gitignore

```diff
@@ -61,6 +61,7 @@ t/
 /dist/
 
 *.memory-card.json
+config/config.yaml
```

Comment:
Could you please explain why you add this ignore, what's the problem if we does not include it?

Reply:
It's a config file for run this project(genarate the registration file or run bridge), containing some personal setting.